src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -81,14 +81,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     QueryResultList<Entity> results = prepQuery.asQueryResultList(fetchOptions);
     ArrayList<Organization> requestedOrganizations = new ArrayList<Organization>();
 
-    /* Fills requestedOrganizations array with 4 fields of each org- name, phone, addres, and desc. */
+    /* Fills requestedOrganizations array*/
     for (Entity entity : results) {
-
-      Organization newOrg = new Organization((String) entity.getProperty("orgName"),
-                                             (String) entity.getProperty("orgEmail"),
+      
+      // TODO(): Implement better schema to represent opening and closing hours for different days
+      long openingHour = (long) ((ArrayList)entity.getProperty("openHours")).get(0);
+      long closingHour = (long) ((ArrayList)entity.getProperty("openHours")).get(1);
+      Organization newOrg = new Organization((long) entity.getKey().getId(), 
```

Comment:
robs comment: This function is getting quite long. How would you start to factor it apart into reusable components?

What do you think of moving this functionality to the Organizations class?

Reply:
We looked at this function and felt that it makes sense to just use the constructor for the organizations class, and that the other tasks are pretty specific to what this servlet does. What are your ideas for ways we can break this servlet up into reusable parts? @mickrobk

Reply:
What are the alternatives you've considered?

Reply:
We considered initializing these objects when we do the Add Organization servlet but we figured that it was be easier to initialize them here since we are getting the entities that we need at this moment rather than searching for the organization object that the entity corresponds to.

We are now considering creating a setter in the organizations class that takes in an entity and sets the member variables according to the entity properties

Reply:
We thought about alternatives some more and realized that we could simplify this function a lot if we create a setter in the Organization class that takes in an entity object and sets all of its properties to the organizations properties. This could be used anywhere we want to populate the fields of an organization object with an entity